client/src/main/java/io/pravega/keycloak/client/PravegaKeycloakCredentials.java

```diff
@@ -26,8 +26,16 @@
 
     // The actual keycloak client won't be serialized.
     private transient KeycloakAuthzClient kc = null;
+    private final String keycloakJsonString;
 
     public PravegaKeycloakCredentials() {
+        init();
+        keycloakJsonString = null;
```

Comment:
I don't think line 33 is needed.

Reply:
Since keycloakJsonString is final (Claudio suggested to made it final), I need to assign a null in this constructor.